tests/datastructures/test_mesh_join_and_weld.py

```diff
@@ -0,0 +1,27 @@
+import copy
+
+from compas.datastructures import Mesh
+from compas.datastructures.mesh.operations.weld import join_meshes
+from compas.datastructures.mesh.operations.weld import weld_mesh
+from compas.datastructures.mesh.operations.weld import join_and_weld_meshes
+
+__author__     = ['Robin Oval']
+__copyright__  = 'Copyright 2018, Block Research Group - ETH Zurich'
+__license__    = 'MIT License'
+__email__      = 'oval@arch.ethz.ch'
+
+mesh_1 = Mesh.from_obj(compas.get('faces.obj'))
+
+mesh_2 = copy.copy(mesh)
+for vkey in mesh_2.vertices():
+    mesh_2.vertex[vkey]['y'] += 10.
+
+mesh_3 = join_meshes([mesh_1, mesh_2])
```

Comment:
These tests will not be picked up by `pytest`, unit tests need to be functions starting with `test_` and should contain `assert` statements to trigger the actual testing. ;)

Check the docs here for more details: https://docs.pytest.org/en/latest/ (or ping me).